src/types/graphql-global-types.ts

```diff
@@ -556,6 +556,8 @@ export enum TeamRole {
   LEARNING = "LEARNING",
   MODEL_LEAD = "MODEL_LEAD",
   MODEL_TEAM = "MODEL_TEAM",
+  OACT = "OACT",
+  PAYMENT = "PAYMENT",
```

Comment:
Looks like the client tests are failing on a snapshot (the dropdown looks to have these new elements auto-added to it)

Should be a quick fix, if you run `scripts/dev test:js[--u]` locally and validate that the right snapshot is updated in the correct way, that should address it!